app/serializers/api/v2/ext_app/deployment_serializer.rb

```diff
@@ -0,0 +1,45 @@
+class ::Api::V2::ExtApp::DeploymentSerializer < ::Api::V2::BaseSerializer
+  def data
+    if @object.is_a? Enumerable
+      {
+        total_server_items: @total_server_items,
+        items: @object.map{ |deployment| serialize(deployment) }
+      }
+    else
+      serialize(@object)
+    end
+  end
+
+  def serialize(deployment)
+    {
+      name: deployment.name,
+      count: deployment.count,
+      definition: deployment.definition,
+      containers: deployment.managed_containers.map{|container| serialize_container(container)}
+    }
+  end
+
+  def serialize_container(container)
```

Comment:
why do we have to create this method? why not just use `serialize` in `https://github.com/pathfinder-cm/pathfinder-mono/blob/master/app/serializers/api/v2/ext_app/container_serializer.rb` ?

Reply:
cc @galuhest.

Reply:
cc @galuhest.

Reply:
it was because we didn't need all column from container mas. But after further discussion, we conclude that the difference is too negligible so we will use the original serializer.